drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.c

```diff
@@ -1318,8 +1319,21 @@ int mt76_connac_mcu_uni_add_bss(struct mt76_phy *phy,
 				.tag = cpu_to_le16(UNI_BSS_INFO_HE_BASIC),
 				.len = cpu_to_le16(sizeof(struct bss_info_uni_he)),
 			},
+			.bss_color = {
+				.tag = cpu_to_le16(UNI_BSS_INFO_BSS_COLOR),
+				.len = cpu_to_le16(sizeof(struct bss_info_uni_bss_color)),
+				.enable = 0,
+				.bss_color = 0,
+			},
```

Comment:
@objelf can you please confirm mt7663 just ignore this field?

Reply:
the UNI_BSS_INFO_BSS_COLOR only supports 80211ax chipset

Reply:
the UNI_BSS_INFO_BSS_COLOR only supports 80211ax chipset

Reply:
what I mean is, is bss_info_uni_bss_color element just skipped by 7663 fw or does it report an error? in the first case the patch is fine